21.0/apache/Dockerfile

```diff
@@ -121,8 +122,8 @@ RUN a2enmod headers rewrite remoteip ;\
     } > /etc/apache2/conf-available/remoteip.conf;\
     a2enconf remoteip
 
-ENV NEXTCLOUD_VERSION 21.0.3
```

Comment:
Using `ARG` to be able to dynamically update the version without changing the Dockerfile.

Reply:
`ARG` cannot be used in official images